amplify/data/resource.ts

```diff
@@ -225,6 +226,23 @@ const schema = a
       .authorization((allow) => [allow.authenticated()])
       .handler(a.handler.function(CreateTeamWithCode)),
 
+    ResetHackathon: a
+      .mutation()
+      .arguments({
+        scoreComponents: a.json(),
```

Comment:
let's mark these all as required. That way you don't need to do ?? stuff elsewhere, minimizing uncertainty